### DISCUSSION TITLE
Pan view / Hand tool

### DISCUSSION THREAD

**Discussion by rafaelaschaan** [uxbox/uxbox#517] [Ideas]
Hey, everyone :) I was just testing Penpot and I have a small feedback, let me try to explain:

I've been using design softwares for a while now, and it is a common behavior to press the space button to move around the screen. I see we have the same feature in Penpot, however with a small difference: to be able to move around, most of this softwares require the user to click with the left button of the mouse (or to pressure your tablet pen, which is my case).

In resume: I'd say it will be easier for designers and users who are already used to this behavior if Penpot has something similar:
1) press space to activate the hand tool
2) click with your mouse to "grab" the screen and
3) let the mouse go when you reach a desired area

How do you guys feel about it? Does it makes sense to you as do to me?

o/

**Comment by Manueljlin** [comment]
I'm honestly used to both, since Penpot has the same behavior as Inkscape but I can see why it's better for a wacom-style device. I think it would be a great idea to add this at least as an option :D

**Reply by rafaelaschaan** [reply]
Haven't work with Inkscape yet, just downloaded to try it out :D 
It could be a matter of usage too, you know, to get used to a new tool

**Comment by elhombretecla** [comment]
Hi all! thank you so much for your feedback :)

It is a very interesting idea to study and make some user testing. We are going to add this to our enhancement list.
